apps/nextjs-website/package.json

```diff
@@ -14,6 +14,8 @@
   "dependencies": {
     "@aws-amplify/auth": "^5.6.6",
     "@aws-amplify/ui-react": "^5.3.1",
+    "@aws-sdk/client-dynamodb": "^3.490.0",
```

Comment:
Since we are adding these new deps, what do you think to use the latest version?